wpeverest/everest-forms
# Feature - Form specific email notification toggle

### All Submissions:






### Changes proposed in this Pull Request:



This PR allows the user to turn on and turn off the email notification.

### How to test the changes in this Pull Request:

1. Select a form and find Email tab under the Settings. 
2. You will see the toggle button.
3. You can turn it on and off by that button.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Form specific email notification toggle.
